docs/advanced-topics/networktime-ticks.md

```diff
@@ -0,0 +1,203 @@
+---
+id: networktime-ticks
+title: NetworkTime and Ticks
+sidebar_label: NetworkTime & Ticks
+---
+
+## LocalTime and ServerTime
```

Comment:
this document is a brand-new add, should this go under a separate PR?

Reply:
it has a seperate PR. this was me not clearing visual studio when I changed the branch and editing this file by mistake. i can remove if its a problem